packages/core/src/browser/icons/spinner.svg

```diff
@@ -0,0 +1,6 @@
+<!--Licensed under the MIT License. See License.txt in the project root for license information.-->
```

Comment:
For my own reference, where does the image come from?
For example:
- what is the original link to the image?
- was the image created manually by you?

Reply:
Idea from [this codepen](https://codepen.io/aleksander351/pen/KzgKPo), implemented myself.